package.json

```diff
@@ -1,24 +1,15 @@
 {
-  "name": "hackmit-reg",
-  "description": "mean stack registration system",
+  "name": "bitcamp-quill",
```

Comment:
Maybe we should do another PR to get rid of all `quill` references and come up with our own project codename since we're doing a pretty hefty re-write ;)
For now maybe `hackathon-reg` or `bitcamp-reg`?

Reply:
Agreed, we should change the name.